setup.py

```diff
@@ -50,7 +50,7 @@
         'testing': [
             'coverage', 'pytest', 'yapf==0.31', 'pytest-cov', 'transformers',
             'pytest-xdist', 'torchvision', 'tabulate', 'efficientnet_pytorch',
-            'pytest-timeout', 'pytorch_geometric'
+            'pytest-timeout', 'torch-geometric'
```

Comment:
this is not the right way to install pyg: https://github.com/pyg-team/pytorch_geometric#pytorch-113

Reply:
@lamyiowce if there is something you can't put in the setup.py, you can always add it to the makefile, which is called by CI: https://github.com/spcl/daceml/blob/master/Makefile#L36